rule/rule_test.go

```diff
@@ -79,7 +84,8 @@ y_library(
     srcs = ["bar.y"],
 )
 
-z(
```

Comment:
Can you add an extra test to confirm that it correctly places `kwargs` after positional `args` even when invoked in the other order?

Reply:
Done